generate_table.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""Script for generation table with data/models from list.json in .md format"""
+import argparse
+import json
+
+
+def generate_table(fn):
+    with open(fn) as infile:
+        data = json.loads(infile.read())
+
+    datasets = sorted(data["corpora"].items(), key=lambda kv: kv[0])
+    models = sorted(data["models"].items(), key=lambda kv: kv[0])
+
+    print("## Available data")
+    print("### Datasets")
+    print("| name | source | description |")
+    print("|------|--------|-------------|")
+    for name, other in datasets:
+        if name.startswith("__testing_"):
+            continue
+        print("| {name} | {source} | {description} |".format(
+            name=name, source=other["source"], description=other["description"]
+        ))
+
+    print("")
+    print("### Models")
+    print("| name | description | papers | preprocessing | parameters |")
+    print("|------|-------------|------------|--------|---------------|")
+    for name, other in models:
+        if name.startswith("__testing_"):
+            continue
+        print("| {name} | {description} | {papers} | {preprocessing} | {parameters} |".format(
+            name=name, description=other["description"], parameters=other.get("parameters", "-"),
+            papers=other.get("papers", "-"), preprocessing=other.get("preprocessing", "-")
+        ))
+    print("(generated by generate_table.py based on lists.json)")
```

Comment:
Shouldn't this be `based on %{fn}` instead?

Reply:
I strongly prefer new `.format`, instead of C-style formatting.

Reply:
About fn - maybe, but file never changed

Reply:
OK, I meant outputting the actual parameter rather hardwiring some "magic" file name.

Btw magic vars are best defined at global scope (`DATASETS_SOURCE = "lists.json"`). It makes it easier to comment / explain / maintain / modify in the future.